src/datascience-ui/ipywidgets/common/kernel.ts

```diff
@@ -314,7 +315,7 @@ class ProxyKernel implements IMessageHandler, Kernel.IKernel {
         this.messageHooks.set(msgId, hook);
 
         // Wrap the hook and send it to the real kernel
-        window.console.log(`Registering hook for ${msgId}`);
+        logMessage(`Registering hook for ${msgId}`);
```

Comment:
Easier to debug & figure out what messages come from the UI & what comes from the extension host.

Reply:
**See here** 
![Screen Shot 2021-04-23 at 09 40 08](https://user-images.githubusercontent.com/1948812/115903731-c66af200-a418-11eb-914f-f9cf8ec69bf0.png)